apps/meteor/.docker/Dockerfile

```
@@ -1,3 +1,7 @@
+ARG DENO_VERSION="1.37.1"
+
+FROM denoland/deno:${DENO_VERSION} as deno
```

I figured they have a bin only variant that suits us better
```suggestion
FROM denoland/deno:bin-${DENO_VERSION} as deno
```